Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -455,6 +470,39 @@ def negMulRelabelling (x y : PGame.{u}) : -x * y ≡r -(x * y) :=
         exact (negMulRelabelling _ _).symm
   termination_by (x, y)
 
+/-- `x * -y` and `-(x * y)` have the same moves. -/
+lemma mul_neg (x y : PGame) : x * -y = -(x * y) :=
```

Comment:
[`mul_neg`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/Ring/Defs.html#mul_neg) is marked as a simp lemma, should this be too?